uli_init/simulate.py

```diff
@@ -43,18 +45,23 @@ def __init__(
         seed=42,
     ):
 
-        self.system = system
-        self.system_pmd = system.system_pmd  # Parmed structure
+        self.system_pmd = system.system  # Parmed structure
         self.r_cut = r_cut
         self.e_factor = e_factor
-        self.tau = tau
+        self.tau_kt = tau_kt
+        self.tau_p = tau_p
         self.dt = dt
         self.auto_scale = auto_scale
         self.ref_units = ref_units
         self.mode = mode
         self.gsd_write = gsd_write
         self.log_write = log_write
         self.seed = seed
+        
+        if nlist.lower() == "cell":
+            self.nlist = hoomd.md.nlist.cell
+        elif nlist.lower() == "tree":
+            self.nlist = hoomd.md.nlist.tree
```

Comment:
```suggestion
        self.nlist = getattr(hoomd.md.nlist, nlist.lower())
```
you can also try this method if you like. It will fail here with attribute error if an incorrect nlist is provided.

Reply:
I like this a lot better.